apps/src/gamelab/GameLab.js

```diff
@@ -85,6 +87,8 @@ const validationLibraryName = 'ValidationSetup';
 const DRAW_LOOP_START = 'drawLoopStart';
 const DRAW_LOOP_MEASURE = 'drawLoop';
 
+const DEFAULT_GENERATED_PROPERTIES = {};
```

Comment:
This seems like an odd way to set a default property. Why use a const for an empty object?
Alternate example:
Delete line 90
Line 114: `this.generatedProperties = {}`
Line 417: `this.generatedProperties = {...initialGeneratedProperties};`

Reply:
Originally, I had properties that I wanted to always include (even if they weren't supplied by the service through `initialGeneratedProperties`). Now, I don't, so this isn't useful

Reply:
I'll remove `DEFAULT_GENERATED_PROPERTIES`

Reply:
👍